docs/reference/openapi-rules.md

```diff
@@ -417,6 +417,7 @@ Server URL should not have a trailing slash.
 ### oas2-operation-security-defined
 
 Operation `security` values must match a scheme defined in the `securityDefinitions` object.
+Ignores empty `security` values.
```

Comment:
@m-mohr From the reader's standpoint, that might be interesting to provide an example of when having an empty value may make sense (as it's not immediately obvious).

Reply:
Done.